### ISSUE TITLE
Version of istio-cni helm chart is not updated

### ISSUE THREAD

**Issue by mcdoker18** [istio/istio#15490]
**Bug description**
All istio helm repositories contain the same version of istio-cni chart.

**Affected product area (please put an X in all that apply)**

[X] Configuration Infrastructure
[ ] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**
A version of istio-cni helm chart is updated for every release of istio-cni.
**Steps to reproduce the bug**
For example:
* https://storage.googleapis.com/istio-release/releases/1.2.2/charts/index.yaml - 0.1.0
* https://storage.googleapis.com/istio-release/releases/1.1.11/charts/index.yaml - 0.1.0

**Comment by howardjohn** [comment]
I believe this is intended and the CNI plugin is considered an addon. Just like we don't ship a Grafana 1.2.2 or 1.1.11 we don't ship CNI 1.2.2.

/assign @rlenglet can you confirm this?

**Comment by sdake** [comment]
@howardjohn it may be considered an addon, however, the helm chart itself needs a version revision such that upgrades work.  Whether it revs to 0.1.1 or 1.2.2 is fine, but it needs a rev, or Helm won't upgrade it.

This might make sense as part of the T&R scripts.

Cheers
-steve

**Comment by howardjohn** [comment]
Good point. It may just make sense to increment it with the rest of istio then. In the future we *might* switch to having all components independent versioning but until then it might not make sense to have it different?

I don't have a strong opinion either way, but once we decide it shouldn't be too hard to fix in the scripts (or manually)

**Comment by sdake** [comment]
@john-a-joyce and @tiswanso seemed to want to stick with version 0.1.0 IIRC, but the version definitely needs a revision on each rev of Istio.

The decision point on separate version numbers for different components is orthogonal at this time to whether or not upgrades should work with CNI.
